indico/modules/rb_new/blueprint.py

```diff
@@ -48,6 +48,7 @@
 _bp.add_url_rule('/api/rooms/<int:room_id>/timeline', 'timeline', bookings.RHTimeline)
 _bp.add_url_rule('/api/rooms/<int:room_id>/stats', 'room_stats', rooms.RHRoomStats)
 _bp.add_url_rule('/api/suggestions', 'suggestions', bookings.RHRoomSuggestions)
+_bp.add_url_rule('/api/rooms/<int:room_id>/check_available', 'check_room_available', rooms.RHCheckRoomAvailable)
```

Comment:
`check-available` or maybe `availability/simple` since it's similar to the 'availability' endpoint just without all the detailed data?

Reply:
move it up one line to group it together with the other `/api/rooms/...` urls